inventory/vagrant/group_vars/webserver/drupal.yml

```diff
@@ -58,6 +58,7 @@ drupal_enable_modules:
 drupal_trusted_hosts:
   - ^localhost$
   - "{{ hostvars[groups['webserver'][0]].ansible_host }}"
+  - "^' . preg_quote($_SERVER['SERVER_NAME']) . '$"
```

Comment:
This seems like a PHP command...does that work within the ansible framework which is (I think) Ruby?

Reply:
I think it works because settings.php is a php file?  I haven't tried it yet to find out.

Reply:
Ohhhhhhh 🤦‍♂

Reply:
Yes, running ansible with this entry added to the variables will add `'^' . preg_quote($_SERVER['SERVER_NAME']) . '$'` to the trusted host patterns. That means any name (or IP) you use to access the Islandora instance in the VM will automagically be trusted, so you don't need to explicitly add it.